spotinst/resource_spotinst_ocean_right_sizing_rule.go

```diff
@@ -402,14 +490,47 @@ func expandLabels(data interface{}) ([]*right_sizing.Label, error) {
 			label.Value = spotinst.String(v)
 		}
 
-		labels = append(labels, label)
-
+		//In case of responseDataLabel empty list not going to validate user's workload again GET attachedWorkload response in IF block.
```

Comment:
modify the statement as below:
In case of responseDataLabel retuned as empty list then not validating user's workload against GET attachedWorkload response.

Reply:
Updated